### PR TITLE
chore: remove redundant Switch component from routing logic

### PR DESCRIPTION
**Pull request by tucker-weed**

### CODE REVIEW

File: app/javascript/components/app.jsx [hunk 1]
```diff
@@ -80,17 +80,15 @@ const App = () => (
   <ApolloProvider client={client}>
     <GlobalStyle />
     <Router>
-      <Switch>
-        <Route
-          path="/:city_name"
```
Comment (tmsavage) [inline app/javascript/components/app.jsx:85]: What is your reasoning for removing `Switch` instead of `Route`?

**Review by cbarber — APPROVED** [review]
lgtm :+1:

File: app/javascript/components/app.jsx (hunk 1 above)
Comment (chewnoill) [reply app/javascript/components/app.jsx:85]: If there is only one `Route`, a `Switch` doesn't really do anything. Kind of like having a [`switch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) with only one `case`

**Review by chewnoill — APPROVED** [review]
🚀

**Pull request merged**